docs/spec/index.html

```diff
@@ -1,740 +1,876 @@
+        href="https://w3id.org/traceability/interoperability/openapi">Open API
+        Specification</a>
+      is designed to be easily implemented and used to access conforming
+      platforms. The following OAS modifications are required:
+    </p>
+    <p>
+      Add the target <code>tokenUrl</code>:
+    <pre class="example">
+        components:
+          securitySchemes:
+            OAuth2:
+              type: oauth2
+              flows:
+                clientCredentials:
+                  tokenUrl: https://conformant-platform.example.com/oauth/token
+        </pre>
+    </p>
+    <p>
+      Add the target <code>server</code>:
+    <pre class="example">
+          servers:
+            - url: https://conformant-platform.example.com
+        </pre>
+    </p>
+  </section>
+
+  <section class="informative">
+    <h2>Privacy Considerations</h2>
+
+    <p>
+      Systems complying with this specification SHOULD leverage and
+      comply with, and service providers complying with this specification
+      SHOULD be able to provide results to, the guidance provided in
+      <a href="https://csrc.nist.gov/publications/detail/sp/800-53/rev-5/final">
+        NIST 800 53 rev 5
+      </a> Security and Privacy Controls for Information Systems and
+      Organizations
+    </p>
+  </section>
+
+  <section class="informative">
+    <h2>Security Considerations</h2>
+
+    <p>
+      There are a number of security considerations that implementers should
+      be aware of when processing data described by this specification.
+      Ignoring or not understanding the implications of this section can
+      result in security vulnerabilities.
+    </p>
+
+    <p>
+      While this section attempts to highlight a broad set of security
+      considerations, it is not a complete list. Implementers are urged to
+      seek the advice of security and cryptography professionals when
+      implementing mission critical systems using the technology outlined in
+      this specification.
+    </p>
+
+    <section class="normative">
+      <h2>General Guidelines</h2>
       <p>
-        See terms as defined in the
-        <a href="https://w3c-ccg.github.io/traceability-vocab/#terminology">
-          Traceability Vocabulary
-        </a>
-        document.
+        As a rule, systems conforming with this specification SHOULD leverage
+        and comply with encryption and security guidelines as listed in:
+        <a href="https://csrc.nist.gov/publications/detail/fips/186/5/draft"
+          rel="nofollow">FIPS 186-5 (DRAFT)</a>,
+        <a href="https://csrc.nist.gov/publications/detail/fips/180/4/final"
+          rel="nofollow">FIPS 180-4</a>, and
+        <a href="https://csrc.nist.gov/publications/detail/fips/197/final"
+          rel="nofollow">FIPS 197</a>. This effectively means that committers
+        should be thinking along the
+        lines of SHA-256, Ed25519, and Rijndael versus other competing
```

Comment:
Is Ed25519 really the right one to focus on here? feels like P-256 / Secp256r1 might be better.

Reply:
agreed - i need to revisit all of these sections from way back with current data